### PR TITLE
Adds counter animation for the formula widget

### PR DESCRIPTION
**Pull request by javierarce**
Original issue: https://github.com/CartoDB/cartodb.js/issues/862

This PR introduces a new method [_animateValue](https://github.com/CartoDB/cartodb.js/pull/864/files#diff-e807cd68943865e365535e446310249dR68) to animate the counters of a widget. The method allows to configure a custom formatter and a template.

animation

animation

Please, can you review this, @xavijam?

### CODE REVIEW

File: src/geo/ui/widgets/formula/animation_template.tpl [hunk 1]
```diff
@@ -0,0 +1 @@
+<%= prefix %><%= value %><%= suffix %>
```
Comment (xavijam) [inline src/geo/ui/widgets/formula/animation_template.tpl]: We can't use <%= %> anymore, remember to use <%- %> instead.

File: themes/scss/widgets/text-elements.scss [hunk 2]
```diff
@@ -6,10 +6,9 @@
 
 .Widget-textBigger { // 16 size
   @include text-overflow();
+  color: #2E3C43;
```
Comment (xavijam) [inline themes/scss/widgets/text-elements.scss]: Don't add colors to the element classes, let's add them to the light theme, please.

Comment (xavijam) [inline themes/scss/widgets/text-elements.scss]: Oh, wait, sorry, you only move the property, @piensaenpixel, for the record :).

Comment (xavijam) [inline themes/scss/widgets/text-elements.scss]: In any case, @javier, could you move that value to light theme please?

File: src/geo/ui/widgets/formula/animation_template.tpl (hunk 1 above)
Comment (javierarce) [inline src/geo/ui/widgets/formula/animation_template.tpl]: Ops, sorry, I had an issue with the separation between prefix, value and suffix, tried several things and forgot to change the template back to the safe setting. It's fixed now.

File: src/geo/ui/widgets/histogram/content-view.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ var WidgetViewModel = require('../widget_content_model');
 var HistogramChartView = require('./chart');
 var placeholder = require('./placeholder.tpl');
 var template = require('./content.tpl');
```
Comment (xavijam) [inline src/geo/ui/widgets/histogram/content-view.js]: Changes over this file are not related with the PR right?

Comment (xavijam) [inline src/geo/ui/widgets/histogram/content-view.js]: Ah, yep, at the end, sorry :joy:.

**Comment by xavijam** [comment]
For me :+1: \o/.

File: themes/scss/widgets/text-elements.scss (hunk 2 above)
Comment (javierarce) [inline themes/scss/widgets/text-elements.scss]: Mmmm… I didn't add that, just moved it to the top so it passed our CSS rules… I'll put it to the light  theme.

Comment (xavijam) [inline themes/scss/widgets/text-elements.scss]: Yep, I realized later -> https://github.com/CartoDB/cartodb.js/pull/864/files#r46034683

**Comment by javierarce** [comment]
Thanks! :heart:
